development/dev-start.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -e
+# we create the kind cluster with some specific configurations, ex: we allow 'host.docker.internal' on the kubeconf
+# we also save the .kubeconfig on the root folder
+kind create cluster --config=./development/kind-config.yml --kubeconfig=./.kubeconfig
+# we make the kubeconfig for the local backend
+cp .kubeconfig .kubeconfig-backend
+# containers reach the host on the 'host.docker.internal' dns instead of 127.0.0.1
+# this line is needed for cross compatibility between osx and unix
+#sed -i.bak 's/127.0.0.1/host.docker.internal/g' .kubeconfig-backend && rm -f .kubeconfig-backend.bak
+sed -i.bak "s/server: $PARTITION_COLUMN.*/server: https:\/\/kind-control-plane:6443/" .kubeconfig-backend && rm -f .kubeconfig-backend.bak
+
+# now that there is the .kubeconfig we can evaluate the .envrc
+source .envrc
```

Comment:
Since `.envrc` is going to be automatically sourced by everybody that have `direnv` installed on their machine and this environment file is supposed to be sourced only after the cluster has started, can we rename the file?